weblate/trans/models/unit.py

```diff
@@ -116,6 +116,17 @@ def same(self, unit, exclude=True):
             result = result.exclude(pk=unit.id)
         return result
 
+    def order_by_request(self, sort_list_request):
+        available_sort_choices = ["-priority", "position", "context", "num_words"]
+        sort_list = []
+        for choice in sort_list_request:
+            if choice in available_sort_choices:
+                sort_list.append(choice)
+        if not len(sort_list):
```

Comment:
Calling len is not needed here:

```suggestion
        if not sort_list:
```